packages/block-editor/src/components/provider/index.js

```diff
@@ -44,6 +47,24 @@ export const ExperimentalBlockEditorProvider = withRegistryProvider(
 		// Syncs the entity provider with changes in the block-editor store.
 		useBlockSync( props );
 
+		const mediaUploadSettings = useMediaUploadSettings( settings );
+
+		if ( window.__experimentalMediaProcessing ) {
+			return (
+				<MediaUploadProvider
+					settings={ mediaUploadSettings }
+					useSubRegistry={ false }
+				>
+					<SlotFillProvider passthrough>
+						{ ! settings?.__unstableIsPreviewMode && (
+							<KeyboardShortcuts.Register />
+						) }
+						<BlockRefsProvider>{ children }</BlockRefsProvider>
+					</SlotFillProvider>
```

Comment:
Nit: we could use a variable to hold the children to avoid "duplicating" the children.